src/test/java/testPages/HomePageTest.java

```diff
@@ -0,0 +1,23 @@
+package testPages;
+
+import base.TestBase;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import steps.HomePageTestSteps;
+
+public class HomePageTest extends TestBase {
```

Comment:
why both tests and steps extend TestBase? it's the same as a chair and a cat will extend parent class Animal, would it be correct?